client/network/src/service.rs

```diff
@@ -855,8 +866,12 @@ struct Metrics {
 	network_per_sec_bytes: GaugeVec<U64>,
 	notifications_queues_size: HistogramVec,
 	notifications_sizes: HistogramVec,
-	num_event_stream_channels: Gauge<U64>,
 	opened_notification_streams: GaugeVec<U64>,
+	out_events_dht_count: Gauge<U64>,
```

Comment:
These would not need to be mentioned here if `ChannelsOut::new` would take a registry.